docs/Configuration.md

```diff
@@ -1278,12 +1278,12 @@ export default config;
 
 #### GitHub Actions Reporter
 
-If included in the list, the built-in GitHub Actions Reporter will annotate changed files with test failure messages:
+If included in the list, the built-in GitHub Actions Reporter will annotate changed files with test failure messages and print logs with github group features for easy navigation. Note that `'default'` should not be used in this case as `'github-actions'` will handle that already, so remember to also include `'summary'`:
 
 ```js tab
 /** @type {import('jest').Config} */
 const config = {
-  reporters: ['default', 'github-actions'],
+  reporters: ['github-actions', 'summary'],
```

Comment:
Hm.. If I get it right, having `reporters: ['github-actions', 'jest-silent-reporter', 'summary']` will not work anymore. I mean, that is current setup of Jest repo: reporter prints dots, annotates files on GitHub and shows summary. If `'github-actions'` is printing logs, there is no way to make silent reporter work together with it.

Reply:
we can add options to skip that? I'm thinking this is fine for now, tho (unless I'm missing something)

Reply:
Added silent option to reporter